src/test/_RomanConverter.java

```diff
@@ -0,0 +1,38 @@
+package test;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import convertion.RomanConverter;
+
+public class _RomanConverter {
+
+	@Test
+	public void romanToDecimal() {
+		int i=RomanConverter.convert("XIII");
+		assertEquals(13, i);
+		i=RomanConverter.convert("IX");
+		assertEquals(9, i);
+		i=RomanConverter.convert("MCMLIV");
+		assertEquals(1954, i); 
+		
+		
+	}
+	
+	@Test
+	public void decimalToRoman() {
+	
+		String i=RomanConverter.convert(5);
+		assertEquals("V", i);
+		i=RomanConverter.convert(10);
+		assertEquals("X", i);
```

Comment:
“have only one logical assertion per test”. 
Remember, unit tests are a design specification of how a certain behaviour should work, not a list of observations of everything the code happens to do.